Gruntfile.js

```diff
@@ -1,99 +1,104 @@
-            },
-            styleguide: {
-                files: {
-                    'css/styleguide.css': 'css/styleguide.scss',
-                },
-            },
-        },
-        watch: {
-            styleguide: {
-                files: ['css/*.scss', 'css/*.hbs', 'css/*.md', 'directives/**/*.js', 'app.js'],
-                tasks: ['sass:styleguide', 'sass:dist','ngtemplates', 'shell', 'cssmin'],
-            },
-        },
-        cssmin: {
-            options: {
-                shorthandCompacting: false,
-                roundingPrecision: -1,
-            },
-            target: {
-                files: {
-                    'css/main.min.css': 'css/main.css',
-                    'css/main_dark.min.css': 'css/main_dark.css'
-                },
-            },
-        },
-        shell: {
-            options: {
-                stderr: false,
-            },
-            target: {
-                command: 'npm run-script kss',
-            },
-        },
-        karma: {
-            styleguide: {
-                configFile: 'karma.conf.js'
-            },
-            continuous: {
-                configFile: 'karma.conf.js',
-                singleRun: false
-            }
-        },
-        jshint: {
-            options: {
-                jshintrc: '.jshintrc' //http://jshint.com/docs/options/
-            },
-            all: ['Gruntfile.js', 'directives/**/*.js', 'app.js'],
-        },
-        jscs: {
-            src: ['Gruntfile.js', 'directives/**/*.js', 'app.js'],
-            options: {
-                config: '.jscsrc',
-            },
-        },
-        ngtemplates: {
-            'styleguide.templates': {
-                src: ['directives/**/*.tpl.html'],
-                dest: 'dist/templates.js',
-                options: {
-                    standalone: true
-                }
-            }
-        },
-        sasslint: {
-                  options: {
-                      configFile: 'config/.sass-lint.yml',
-                  },
-                  target: ['location/*.scss']
-              },
-        uglify: {
-            'dist/wfmdirectives.min.js':
-            ['directives/**/*.js', '!directives/**/*.spec.js']
-        }
-    });
+    grunt.initConfig({
+       sass: {
+           dist: {
+               files: {
+                   'css/main.css': 'css/main.scss',
+                   'css/main_dark.css': 'css/main_dark.scss'
+               },
+           },
+           styleguide: {
+               files: {
+                   'css/styleguide.css': 'css/styleguide.scss',
+               },
+           },
+       },
+       watch: {
+           styleguide: {
+               files: ['css/*.scss', 'css/*.hbs', 'css/*.md', 'directives/**/*.js', 'app.js'],
+               tasks: ['sass:styleguide', 'sass:dist','ngtemplates', 'shell', 'cssmin'],
+           },
+           test:{
+               files:['directives/**/*.js'],
+               tasks:['karma:continuous']
```

Comment:
Dont we need to add the uglification task here aswell? otherwise only the tests will run new code but not the js